mono/metadata/w32mutex-unix.c

```diff
@@ -380,6 +380,8 @@ ves_icall_System_Threading_Mutex_ReleaseMutex_internal (gpointer handle)
 	pthread_t tid;
 	gboolean ret;
 
+	HANDLE_FUNCTION_ENTER ();
```

Comment:
There are a couple of places inside `ReleaseMutex_internal` where it exits early using `return`.  Those need to change - wrapping a function in `HANDLE_FUNCTION_` means that it has to be single-exit.

Reply:
thanks, updated!  Let me know if we have proper macros for it, I'm a bit lost in that world 🙂